src/components/filters/MinMaxFilter/index.tsx

```diff
@@ -68,6 +86,34 @@ export const MinMaxFilter: React.FC<Props> = (props) => {
     [onChangeMin]
   )
 
+  const clearMin = React.useCallback(() => {
+    if (minInput.current) minInput.current.value = ''
+    setMin(null)
+  }, [minInput])
+
+  const clearMax = React.useCallback(() => {
+    if (maxInput.current) maxInput.current.value = ''
+    setMax(null)
+  }, [maxInput])
+
+  React.useEffect(() => {
+    if (minFromProps === null) {
+      clearMin()
+    }
+  }, [minFromProps, clearMin])
+
+  React.useEffect(() => {
+    if (maxFromProps === null) {
+      clearMax()
+    }
+  }, [maxFromProps, clearMax])
+
+  const clear = React.useCallback(() => {
+    clearMin()
+    clearMax()
+    onClear && onClear()
+  }, [clearMin, clearMax, onClear])
```

Comment:
This is very similar to date, just added some `useRef`. Because is cleared from outside and from inside it needs and effect to react to props changes but only if that change is to make the value null.